hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/featurestore/featuregroup/FeaturegroupController.java

```diff
@@ -123,44 +123,20 @@ private Connection initConnection(String databaseName, Project project, Users us
           "keyStorePassword=" + password;
 
       return DriverManager.getConnection(jdbcString);
```

Comment:
`DriverManager.getConnection(jdbcString)` can throw SQLException. You should either catch it here, `removeCertificatesLocal` and throw propagate the exception, or in `executeReadHiveQuery` and `executeUpdateHiveQuery`  call `initConnection` inside the the try...catch block. If an SQLException occurs, the finally block will remove the references.